octopoes/tests/integration/test_nibbles.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+from datetime import datetime
+from unittest.mock import Mock
+
+import pytest
+from nibbles.definitions import NibbleDefinition, NibbleParameterDefinition
+
+from octopoes.core.service import OctopoesService
+from octopoes.models import OOI, ScanLevel
+from octopoes.models.ooi.network import Network
+
+if os.environ.get("CI") != "1":
+    pytest.skip("Needs XTDB multinode container.", allow_module_level=True)
+
+NMAX = 13
```

Comment:
Can we use a more descriptive name here? I'm guessing it has to do with the network name length, so maybe `MAX_NAME_LENGTH`? (or `MAX_NETWORK_NAME_LENGTH`)